jenkins/inteledison_c.sh

```diff
@@ -2,7 +2,7 @@
 # Copyright (c) Microsoft. All rights reserved.
 # Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-build_root=$(cd "$(dirname "$0")/.." && pwd)
+build_root=$(cd "$(dirname "$0")/../v1" && pwd)
```

Comment:
I wonder if it's easier and/or cleaner to just point the Jenkins jobs to v1/jenkins/... instead of promoting the jenkins/ folder back back up to the repo root?

Reply:
I couldn't do that initially, but I have access now, so I can revert the change.